lib/processImage.js

```diff
@@ -113,13 +113,9 @@ module.exports = (options) => {
         }
       }
       delete req.headers['if-modified-since']; // Prevent false positive conditional GETs after enabling processimage
-      // hijackResponse will never pass an error here
-      // eslint-disable-next-line handle-callback-err
-      hijackResponse(
+            hijackResponse(
```

Comment:
Is there something up with the indentation here?

Reply:
I fixed that in a commit just as you submitted that review :-) https://github.com/papandreou/express-processimage/pull/120/commits/c39ba2b28ab4513cf15e03876ebcabf775549c02